index.html

```diff
@@ -44,7 +46,163 @@ <h2 class="heading-2">I’m a software developer</h2>
+                                  components</h3>
+                                <ul class="btns" id="foo">
+                                </ul>
+                                <p class="popup-text font-f"></p>
+                                <div class="popup-btn-container">
+                                  <a href="" class="popup-btn font-f see-live">See Live<span><i class="fa fa-external-link"></i></span></a>
+                                  <a href="" class="popup-btn font-f see-source">See Source<span><i class="fa fa-github"></i></span></a>
+                                </div>
+                              </div>
+                        </div>
+                      </div>
+                </section>
+                <section class="tech">
+                    <ul class="tech-list">
+                        <li class="tech-btn">
+                            <img src="./img/icon-languages.svg" alt="logo">
+                            <h2 class="lang">Languages</h2>
+                            <ul class="card-list">
+                                <li class="buttons">JavaScript</li>
+                                <li class="buttons">Ruby</li>
+                                <li class="buttons">Html</li>
+                                <li class="buttons">Css</li>
+                            </ul>
+                        </li>
+                        <li class="tech-btn">
+                            <img src="./img/icon-frameworks.svg" alt="logo">
+                            <h2 class="lang">Frameworks</h2>
+                            <ul class="card-list">
+                                <li class="buttons">Bootstrap</li>
+                                <li class="buttons">Ruby on rails</li>
+                                <li class="buttons">RSpec</li>
+                                <li class="buttons">Capybara</li>
+                                <li class="buttons">Selenium</li>
+                            </ul>
+                        </li>
+                        <li class="tech-btn">
+                            <img src="./img/icon-skills.svg" alt="logo">
+                            <h2 class="lang">Skills</h2>
+                            <ul class="card-list">
+                                <li class="buttons">Codekit</li>
+                                <li class="buttons">GitHub</li>
+                                <li class="buttons">Codepen</li>
+                                <li class="buttons">Gitlab</li>
+                                <li class="buttons">Terminal</li>
+                            </ul>
+                        </li>
+                    </ul>
+                </section>
+            </div>
+        </section>
+               
+        <script src="main.js"></script>
+
+    </main>
     <script src="https://use.fontawesome.com/84bcd6cf83.js"></script>
 <script src="main.js"></script>
```

Comment:
- [ ] [OPTIONAL] In this project you are not expected to add any javascript code yet unless specified in the project requirement. Kindly remove the above-selected line of code.